app/lib/features/invite_members/widgets/space_member_invite_card.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:acter/common/models/profile_data.dart';
+import 'package:acter/common/providers/room_providers.dart';
+import 'package:acter/common/providers/space_providers.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:skeletonizer/skeletonizer.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+class SpaceMemberInviteCard extends ConsumerWidget {
+  final Space space;
+  final bool isSelected;
+  final ValueChanged<bool?> onChanged;
+
+  const SpaceMemberInviteCard({
+    super.key,
+    required this.space,
+    this.isSelected = false,
+    required this.onChanged,
+  });
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final profile = ref.watch(spaceProfileDataProvider(space));
+
+    return profile.when(
+      data: (profile) => _spaceItemUI(context, ref, profile),
+      error: (error, stack) => ListTile(
+        title: Text(
+          L10n.of(context).errorLoadingSpace(error),
+        ),
+      ),
+      loading: () => Skeletonizer(
+        child: ListTile(
+          title: Text(space.getRoomIdStr()),
+          subtitle: Text(L10n.of(context).loading),
+        ),
+      ),
+    );
+  }
+
+  Widget _spaceItemUI(
+    BuildContext context,
+    WidgetRef ref,
+    ProfileData profile,
+  ) {
+    final roomId = space.getRoomIdStr();
+    final title = profile.displayName?.isNotEmpty == true ? profile.displayName! : roomId;
+    final parent = ref.watch(canonicalParentProvider(roomId)).valueOrNull;
+    final parentRoomId = parent?.space.getRoomIdStr();
+
+    final avatar = ActerAvatar(
```

Comment:
Don't we already have a widget already doing all this?

Reply:
Yes true but don't wanted to make existing widget to complex as this one contains trailing checkbox as well as members counts as sub-title. Event in future if we want to do any customisation with respect to space member invite design then we can use this separate card.